### ISSUE TITLE
It still has the bug ,when you die you respawn without the skin (also the same if you put skins to your allies)

### ISSUE THREAD

**Issue by DeyuCookie** [hydy100/R3nzSkin#87]
.

**Comment by kirites** [comment]
If you hide the main interface, you will encounter this problem, if you don't hide this problem will be solved

**Comment by DeyuCookie** [comment]
> If you hide the main interface, you will encounter this problem, if you don't hide this problem will be solved

So basically i need to keep the app on the screen while i play ;/ ?

**Comment by kirites** [comment]
> > 如果你隐藏主界面，你会遇到这个问题，如果你不隐藏这个问题就会得到解决
> 
> 所以基本上我需要在玩游戏时将应用程序保持在屏幕上 ;/ ？
Yes, we can solve this problem in this way before the author updates.

**Comment by kirites** [comment]
> > If you hide the main interface, you will encounter this problem, if you don't hide this problem will be solved
> 
> So basically i need to keep the app on the screen while i play ;/ ?

Yes, we can solve this problem in this way before the author updates.

**Comment by DeyuCookie** [comment]
Thank you so much for the tip <3
